website/blog/2023-09-14-closed-source-does-not-equal-secure/item.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Closed source does not equal secure
+slug: 2023-09-14-closed-source-does-not-equal-secure
+authors:
+    - name: Jens Langhammer
+      title: CTO at Authentik Security Inc
+      url: https://github.com/BeryJu
+      image_url: https://github.com/BeryJu.png
+tags:
+    - open core
+    - SSO
+    - open source
+    - community
+    - identity provider
+    - enterprise
+    - source available
+    - closed source
+    - security
+    - authentication
+hide_table_of_contents: false
+image: ./image1.jpg
+---
+
+> **_authentik is an open source Identity Provider that unifies your identity needs into a single platform, replacing Okta, Active Directory, and auth0. Authentik Security is a [public benefit company](https://github.com/OpenCoreVentures/ocv-public-benefit-company/blob/main/ocv-public-benefit-company-charter.md) building on top of the open source project._**
+
+---
+
+Legacy security vendors that rely on black box development are no match for the pace of innovation and transparency of open source. It's an oft-discussed topic, the ability of open source communities to quickly jump in and collectively solve problems and innovate solutions, but it is equally believed that "serious" security software companies have proprietary software.
+
+In this blog, we will take a closer look at the pros and cons of the various source availability types of SSO and other security software.
+
+!["mike-kononov-lFv0V3_2H6s-unsplash.jpg"](./image1.jpg)
+
+<!--truncate-->
+
+Since we’re going to use these terms a lot in our discussion, this is what we mean by open source, closed source, and so on:
```

Comment:
This differs from the most recent version in the Google doc

Reply:
Ah, only the table formatting is different, not the words/definitions. The formatting from the google doc translated into a table with a header row, which we don't want.